### ISSUE TITLE
Make it easier to track confirmations of a deposit

### ISSUE THREAD

**Issue by pdyraga** [threshold-network/token-dashboard#429]
Right now, the user who deposited Bitcoin does not see the number of Bitcoin confirmations on their deposit transaction and does not know how long it is going to take for the optimistic minting to be requested.

This requires design and implementation.

**Comment by sashatanase** [comment]
Following our discussion and decision today, we will add the BTC Confirmations loader on the Success Page like here. 


![Image](https://user-images.githubusercontent.com/40768736/217020884-5b3e6646-5b26-4d43-8f96-17dc56004bbb.png)

BTC Confirmations loader

The Loader should be dynamic and fill each BTC confirmation. In the end, when it reaches 6/6, the checkmark will turn from grey to purple.

Behaviour Prototype Animation 

https://www.figma.com/proto/Tx1lZc7us4SqCeAgbt1hBV/Threshold-Dapp?page-id=7126%3A198919&node-id=7919%3A208962&viewport=-1672%2C-20428%2C1.12&scaling=min-zoom&starting-point-node-id=7918%3A208894&show-proto-sidebar=1


![Image](https://user-images.githubusercontent.com/40768736/217024396-d1e519ef-f66c-4a7b-b647-32b4ff957d9d.png)



Figma Files

https://www.figma.com/file/Tx1lZc7us4SqCeAgbt1hBV/Threshold-Dapp?node-id=7846%3A206304&t=DPkqsh0XagqqVP3k-1

**Comment by sashatanase** [comment]
Please review the following copy. 

```
The tBTC minting process will start only after 6 Bitcoin network confirmations.
```

**Comment by pdyraga** [comment]
Here are the rules:
- If the amount is less than 0.1 BTC, it should have at least 1 confirmation. 
- If the tx is less than 1 BTC, it should have at least 3 confirmations.
- Otherwise, we need to wait for 6 confirmations.

**Comment by mhluongo** [comment]
Note that I want to start the minting request even earlier, but that won't change for at least one maintainer release. Until then, @pdyraga 's rules are correct.

https://github.com/keep-network/optimistic-minting/issues/65

**Comment by mhluongo** [comment]
Closed by #444
